Added an advisory in the Things to Remember content to not include <f…

jjgriff93 opened the pull request:
…orm> tags in custom UI

<form> tags cause the POST from the form submissions of B2C injected pages to not post to the correct endpoint, and it exposes the clients credentials that they've entered in the form. It may not immediately be obvious to people what's happening here so this may help them avoid these scenarios.

azurecla commented:
Hi __@brianbruff__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@brianbruff find a code bug, win a prize! :trophy: Seriously, thanks for finding and fixing this. We really appreciate your help improving the (code) quality of the Azure docs.

Commit:
Added an advisory in the Things to Remember content to not include <form> tags in custom UI

<form> tags cause the POST from the form submissions of B2C injected pages to not post to the correct endpoint, and it exposes the clients credentials that they've entered in the form. It may not immediately be obvious to people what's happening here so this may help them avoid these scenarios.
